src/miner.js

```diff
@@ -1,7 +1,7 @@
 require('dotenv').config();
 const hashing = require('../src/hashing');
-import { getBlockchain } from './blockchain';
-import { Block } from '../src/Block';
+// import { getBlockchain } from './blockchain';
+// import { Block } from '../src/Block';
```

Comment:
No commented out code please, if you don't need something just remove it https://en.wikipedia.org/wiki/You_aren%27t_gonna_need_it